### PR TITLE
Change Menu Options

### PR DESCRIPTION
**Pull request by tbosch82**
Changing/adding menu options as needed. 

### Add Dashboard
dropmenu1

### CODE REVIEW

**Comment by tbosch82** [comment]
Tabs for trading and research have been added. Account drop down control also added. Select boxes, input boxes, and date picker themes and styling have been adjusted to fit overall theme and style.

### New Layout
New Layout

**Comment by tbosch82** [comment]
@ericdudley take a look and let me know what you think.

File: demo/src/Dashboard/Dashboard.scss [hunk 1]
```diff
@@ -21,8 +27,33 @@
     width: 100%;
     display: flex;
     flex: 1;
+    border-bottom: 2px solid $primary;
+
+    &__tabs {
+      background-color: $white !important;
```
[the remark this answers is not in the document]
Comment (tbosch82) [reply demo/src/Dashboard/Dashboard.scss]: Created `TabButton`

**Comment by ericdudley** [comment]
Looks great, feel free to merge, and then I can integrate my PR.

**Comment by tbosch82** [comment]
@ericdudley I made some changes. I am having a little trouble with `react-router`. How do I use it to keep the state of the tabs? I have it working now by going to the trading screen on dashboard creation or dashboard changes, how can i persist the selected tab across dashboards?

**Pull request merged**